src/peft/peft_model.py

```diff
@@ -336,6 +338,33 @@ def from_pretrained(
         else:
             raise ValueError(f"The input config must be a PeftConfig, got {config.__class__}")
 
```

Comment:
maybe add a check for `hf_device_map` here and that we indeed that modules offloaded to `disk`